.github/workflows/awsfulltest.yml

```diff
@@ -14,19 +14,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: nf-core/tower-action@master
+        uses: nf-core/tower-action@v2
         with:
           workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
-          bearer_token: ${{ secrets.TOWER_BEARER_TOKEN }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
           compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
           pipeline: ${{ github.repository }}
-          revision: ${{ github.sha }}
-          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/mhcquant/work-${{ github.sha }}
-          # Add full size test data (but still relatively small datasets for few samples)
-          # on the `test_full.config` test runs with only one set of parameters
-          # Then specify `-profile test_full` instead of `-profile test` on the AWS batch command
+          revision: ${{ github.sha } }
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/mhcquant/work-${{ github.sha }}
           parameters: |
             {
-              "outdir" : "s3://${{ secrets.AWS_S3_BUCKET }}/mhcquant/results-${{ github.sha }}",
+              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/mhcquant/results-${{ github.sha }}"
             }
-          profiles: '[ "test_full", "aws_tower" ]'
+          profiles: test,aws_tower
+          pre_run_script: 'export NXF_VER=21.10.3'
```

Comment:
The CI nextflow version and this one don't match. i am not sure it is a big deal, but to make you aware of this

Reply:
@FriederikeHanssen I also don't know if it would be a problem, but I might change it in the CI then 😄